### ISSUE TITLE
Check for NuGet feed?

### ISSUE THREAD

**Issue by jamesmontemagno** [Redth/dotnet-maui-check#31]
One common issue is that people don't have the nuget feed setup. Can we do that?

**Comment by HenJigg** [comment]
Met this problem

**Comment by HenJigg** [comment]
You can try this operation to check whether the address of maui-preview is correctly referenced in nuget.config

nuget.config:  
<add key="maui-preview" value="https://aka.ms/maui-preview/index.json" />

Add the following NuGet feed to get access to the .NET MAUI packages:
dotnet nuget add source -n maui-preview https://aka.ms/maui-preview/index.json

**Comment by HenJigg** [comment]
Because of the problem caused by nuget.config, modify maui-preview in nuget.config to solve

**Comment by jamesmontemagno** [comment]
I know it is in the getting started guide, but a bit tricky to find https://github.com/dotnet/maui/wiki/Getting-Started

**Comment by ncarandini** [comment]
Instead of using the suggested CLI commands for each new MAUI project, woudn't be better to add the source directly to the Visual Studio? It would be really nice if maui-check could suggest it and do it on user approval.
I did it by myself and (as expected) it works fine for every MAUI project I work with:

![image](https://user-images.githubusercontent.com/139274/120101452-78978700-c146-11eb-831d-bdc392163c4c.png)

**Comment by jamesmontemagno** [comment]
Yeah, we should add it into the wiki now with VS support dropping, previously it was only only cli
